tasks/server-tests/vitest.setup.mts

```diff
@@ -0,0 +1,130 @@
+import { fileURLToPath } from 'node:url'
+
+import { afterAll, afterEach, beforeAll, expect } from 'vitest'
+import { fs, path, $ } from 'zx'
+import type { ProcessPromise } from 'zx'
+
+import { getConfig } from '@redwoodjs/project-config'
+
+$.verbose = !!process.env.VERBOSE
+
+type TestContext = {
+  p?: ProcessPromise
+  projectConfig: ReturnType<typeof getConfig>
+}
+export const testContext: TestContext = {
+  // Casting here because `beforeAll` below sets this and this file runs before all tests.
+  // Working around it being possibly undefined muddies the code in the tests.
```

Comment:
```suggestion
  // Casting here because `beforeAll` below sets this and this file runs before all tests.
  // Working around it being possibly undefined muddies the code in the tests.
  // Also can't just call `getConfig()` because RWJS_CWD hasn't been set yet
```